Tutorial/Solutions.lean

```diff
@@ -113,13 +113,12 @@ theorem list_nth_mut1_spec {T: Type} [Inhabited T] (l : CList T) (i : U32)
         simp only [hiEq]
       . -- Backward function
         intro x'
-        progress as ⟨ tl2, htl2 ⟩
-        simp
         have hUpdate := List.update_nzero_cons hd tl.toList i.toNat x' (by scalar_tac)
         simp only [hUpdate]
-        simp only [htl2]
-        have hiEq : i1.toNat = i.toNat - 1 := by scalar_tac
-        simp only [hiEq]
+        -- simp at ih
+        have hiEq : (i1: ℤ).toNat = i.toNat - 1 := by scalar_tac
+        rw [← hiEq]
```

Comment:
Can you use `simp` or `simp only` instead of `rw`? I'd rather not use too many different tactics in the tutorial (at least at the beginning - of course we need `rw` to unfold the definitions at the beginning of the proof, but I would like to keep only this).

Reply:
Done in 5498a09